First stab at a charter for a WebView CG (#1)
WebView-CG/charter: charter.md

# [DRAFT] WebViews Community Group Charter

> This Charter is work in progress. To submit feedback, please use [the GitHub repo](https://github.com/WebView-CG/charter) Issues where this chharter is being developed.

Reviewer: ```suggestion
This Charter is work in progress. To submit feedback, please use [the GitHub repo](https://github.com/WebView-CG/charter) Issues where this charter is being developed.
```

## Goals

> The WebViews Community Group aims to reduce the issues arising from the use of software components (typically refered as `WebView`s) that are used to render Web technology-based content outside of a Web browser.

Author: ```suggestion
The WebViews Community Group aims to reduce the issues arising from the use of software components (typically referred as `WebView`s) that are used to render Web technology-based content outside of a Web browser.
```

> The WebViews Community Group aims to reduce the issues arising from the use of software components (typically refered as `WebView`s) that are used to render Web technology-based content outside of a Web browser.

Reviewer: - "refered" with the HTTP header? ;)
- Is it only about rendering? Are runtimes such as Deno and Node.js completely out of scope of the first phase?

The notion of "embedded web" was used during today's conversation. I wonder whether we could use it.

```suggestion
The WebViews Community Group aims to reduce issues arising from the use of software components (typicallly referred to as `WebView`s) that allow applications to embed Web technologies outside of a Web browser.
```

Author: I only put one "r" in referred, but you put 3 "l" in typically :)

I think we should limit the scope to WebViews (and thus rendering) - notwithstanding the broader web-native conversation we also want to start

## Scope of Work

In the first phase of work that this charter covers, the WebViews Community Group will:
* identify representative use cases of where WebViews are being used,
* identify the issues that arise from these usages,

> * suggest if these issues can be addressed through improvements to the Web Platform, improvements to its surrounding ecosystem (e.g. documentation, testing frameworks) or through other mechanisms,

Reviewer: A bit hard to parse

```suggestion
* determine whether these issues can be addressed through improvements to the Web Platform, the surrounding ecosystem (e.g. documentation, testing frameworks) or through other mechanisms,
```

> * raise awareness of the impact of WebViews in usage of Web technologies beyond Web browsers in relevant communities (e.g. existing W3C groups).

Reviewer: Same here, suggest to simplify to:
```suggestion
* raise awareness about the impact of WebViews for Web technologies in relevant communities (e.g. existing W3C groups)
```

The group will not publish Specifications on topics other than those listed under Specifications above. See below for [how to modify the charter](#amendments-to-this-charter).

### Out of Scope

> Specific development of technical proposals to mitigate these issues are out of scope of this charter; the group expects to [amend its charter](#amendments-to-this-charter) with a specific list of mitigations it would seek to address in a second phase of work, based on the analysis conducted as described above.

Reviewer: ```suggestion
Specific development of technical proposals to mitigate identified issues are out of scope of this charter; the group expects to [amend its charter](#amendments-to-this-charter) with a specific list of mitigations it would seek to address in a second phase of work, based on the analysis conducted as described above.
```
